res/config/config.php

```diff
@@ -34,6 +35,10 @@
     'debug' => false,
     'path.cache' => __DIR__ . '/../../var/cache',
     'path.logs' => __DIR__ . '/../../var/log',
+    'path.public' => __DIR__ . '/../../web',
+
+    // Platform.sh defines this variable
+    'http.port' => env('PORT', 8000),
```

Comment:
By default the application will run on port `8000` (e.g. in dev environment when running `make preview`).